roles/docker_storage_driver/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+- openshift_facts
```

Comment:
I'm hoping to refactor all of the docker portions out of openshift_facts.  That almost shipped this cycle, but slipped due to feature freeze.